### ISSUE TITLE
Build is failing (missing key?)

### ISSUE THREAD

**Issue by seanfcarroll** [relrod/bing_translator-gem#37]
The CI is failing on every test.

```
  1) BingTranslator translates text
     Failure/Error: result = translator.translate message_en, :from => :en, :to => :ru
     BingTranslator::Exception:
       (s:Client) ArgumentException: The received token is of incorrect token type. : ID=0243.V2_Soap.Translate.440CE7CC
     # ./lib/bing_translator.rb:152:in `rescue in result'
     # ./lib/bing_translator.rb:150:in `result'
     # ./lib/bing_translator.rb:38:in `translate'
     # ./spec/bing_translator_spec.rb:19:in `block (2 levels) in <top (required)>'
```

Is the Bing key missing from the CI or is there some other issue?

**Comment by seanfcarroll** [comment]
Hi @abrisse I was wondering if this would be fixed? I think a key just needs to be added to the CI.

Cheers!

**Comment by relrod** [comment]
I fixed this a few days ago, thanks for the report.

**Comment by relrod** [comment]
I *attempted* to fix this a few days ago, thanks for the report... It's still broken, but I haven't had time to figure out the fix yet. It fails in the same way when I run the tests with `bundler exec rake` but just doing `rake` or `rspec .` works fine...

**Comment by relrod** [comment]
Fixed with #39 hopefully
